### DISCUSSION TITLE
Using another dataset

### DISCUSSION THREAD

**Discussion by Marie0704** [Project-MONAI/tutorials#1581] [Q&A]
Good morning,

I wanted to ask about the possibility of using another dataset. I tried to use the one from Task3, i.e. liver images. However, there is no fixed pair there. Can I ask for a hint on how to generate a pair of constants (I care about supervised learning) for Task 3 so that this dataset can be used for model training?
Can I ask for a documentation proposal that describes quite precisely and extensively what is happening inside LocalNet?
Should the moving and fixed slices be matched to each other, or does it not matter that, for example, the moving photo started much higher (e.g. you can see the end of the thoracic spine) while the moving pair starts already in the lumbar section?

**Comment by KumoLiu** [comment]
Hi @Marie0704, thanks for your interest here.
- If you want to generate some pairs yourself, you can try using some augmentation, such as Rotate.
> Can I ask for a documentation proposal that describes quite precisely and extensively what is happening inside LocalNet?

- I would suggest you refer to the original paper.
https://github.com/Project-MONAI/MONAI/blob/8e134b8cb92e3c624b23d4d10c5d4596bb5b9d9b/monai/networks/nets/regunet.py#L403

> Should the moving and fixed slices be matched to each other, or does it not matter that, for example, the moving photo started much higher (e.g. you can see the end of the thoracic spine) while the moving pair starts already in the lumbar section?

- The moving and fixed slices should be matched. The purpose must be that you want to predict from moving data to fixed data, how will the mismatch be predicted.

Hope it helps, thanks!
